### PR TITLE
New port: (Savant Ascent)

### PR DESCRIPTION
**Pull request by JanTrueno**
# New Port for { Savant Ascent }

## Game Information
- **Title**: { Savant Ascent }
- **URL**: { https://store.steampowered.com/app/259530/Savant__Ascent/ }

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [X] AmberELEC
- [X] ArkOS
- [X] JELOS

### Resolution Tests
Test all major resolutions:
- [X] 480x320
- [X] 640x480
- [X] 720x720 (RGB30)
- [X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/savant/Savant Ascent.sh [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
+get_controls
+
+# Declare variables
+GAMEDIR="/$directory/ports/savant"
+
+# Exports
+if [ "$OS_NAME" == "JELOS" ]; then
```
Comment (kloptops) [inline ports/savant/Savant Ascent.sh:25]: Change this to use the mod_${CFW_NAME}.

Below source device_info.txt
```bash
source $controlfolder/device_info.txt

[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
```

**Review by kloptops — CHANGES_REQUESTED** [review]
Just need to change this bit, otherwise looks good.

**Review by Cebion — CHANGES_REQUESTED** [review]
if [ -f "/etc/os-release" ]; then
  source "/etc/os-release"
fi 

Is not used since that is in device_info already
also Screenshot needs to show gameplay not the title.

**Comment by JanTrueno** [comment]
- Fixed .sh file
- Different screenshot to show gameplay

Thanks for the feedback guys :)

**Review by Cebion — CHANGES_REQUESTED** [review]
We don't use 
```
if [ "$OS_NAME" == "JELOS" ]; then
export SPA_PLUGIN_DIR="/usr/lib32/spa-0.2"
export PIPEWIRE_MODULE_DIR="/usr/lib32/pipewire-0.3/"
fi 
```
This is now in the mod_jelos file and is called `[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"`

**Review by kloptops — APPROVED** [review]
👍

**Pull request merged**